.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "semistandard"
+}
```

Comment:
Not exactly sure, but it makes my Atom editor eslint plugin happy. Cool kids like girder have one :)